src/custom_nodes/east_ocr/Dockerfile.centos

```diff
@@ -19,9 +19,14 @@ RUN yum update -y && yum install centos-release-scl -y && yum install -y devtool
 RUN curl -s https://download.01.org/opencv/master/openvinotoolkit/thirdparty/linux/opencv/opencv_4.5.1-044_centos7.txz | tar --use-compress-program=xz -xvf -
 ARG NODE_NAME=east_ocr
 
+COPY ./queue.hpp ./queue.hpp
```

Comment:
we should drop centos images

Reply:
should we get rid of all `Dockerfile.centos` files in custom nodes?

Reply:
yes

Reply:
completely dropped centos images